Update response fields for listing builds (#1340)
zeit/docs: pages/docs/api/v2/api-docs-mdx/endpoints/deployments.mdx

| Key              | <HelpLink href="#api-basics/types" hasIcon>Type</HelpLink> | Description                                                                                                                                                                                               |
| ---------------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **id**           | <HelpLink href="#api-basics/types">ID</HelpLink>           | The unique identifier of the Build.                                                                                                                                                                       |
| **deploymentId** | <HelpLink href="#api-basics/types">ID</HelpLink>           | The unique identifier of the deployment.                                                                                                                                                                  |

> | **entrypoint**   | <HelpLink href="#api-basics/types">String</HelpLink>       | The entrypoint path of the deployment.                                                                                                                                                                    |

Reviewer: ```suggestion
| **entrypoint**   | <HelpLink href="#api-basics/types">String</HelpLink>       | The entrypoint of the deployment.                                                                                                                                                                    |
```

| **readyState**   | <HelpLink href="#api-basics/types">Enum</HelpLink>         | The state of the deployment depending on the process of deploying, or if it is ready or in an error state. Possible values are `INITIALIZING`, `ANALYZING`, `BUILDING`, `DEPLOYING`, `READY`, or `ERROR`. |
| **readyStateAt** | <HelpLink href="#api-basics/types">Date</HelpLink>         | The time at which the Build state was last modified.                                                                                                                                                      |

> | **scheduledAt**  | <HelpLink href="#api-basics/types">Date</HelpLink>         | The time at which the Build was scheduled for building.                                                                                                                                                   |

Reviewer: ```suggestion
| **scheduledAt**  | <HelpLink href="#api-basics/types">Date</HelpLink>         | The time at which the Build was scheduled to be built.                                                                                                                                                   |
```

| **createdAt**    | <HelpLink href="#api-basics/types">Date</HelpLink>         | The time at which the Build was created.                                                                                                                                                                  |

> | **createdIn**    | <HelpLink href="#api-basics/types">String</HelpLink>       | The region where the Build was first created, e.g. `sfo1`.                                                                                                                                                |

Reviewer: ```suggestion
| **createdIn**    | <HelpLink href="#api-basics/types">String</HelpLink>       | The region where the Build was first created, for example,`sfo1`.                                                                                                                                                |
```

Reviewer 2: I think there is a space missing between `,` and `sfo`

| **use**          | <HelpLink href="#api-basics/types">String</HelpLink>       | The Builder the Build used to generate the output.                                                                                                                                                        |
| **config**       | <HelpLink href="#api-basics/types">Map</HelpLink>          | An object that contains the Build's configuration. For example, <InlineCode>{ "zeroConfig": true }</InlineCode>.                                                                                          |

> | **output**       | <HelpLink href="#api-basics/types">List</HelpLink>         | List of outputs for the Build. These can either be serverless functions (lambdas) or static files.                                                                                                        |

Reviewer: ```suggestion
| **output**       | <HelpLink href="#api-basics/types">List</HelpLink>         | A list of outputs for the Build. These can either be Serverless Functions or static files.                                                                                                        |
```

| **fingerprint**  | <HelpLink href="#api-basics/types">String</HelpLink>       | If the Build uses the `@now/static` Builder, it contains a hashed string of all outputs.                                                                                                                  |

##### Build Output Key

> These are the keys of the `output` key of the Build.

Reviewer: ```suggestion
These are the keys of the `output` Map of the Build.
```

| Key        | <HelpLink href="#api-basics/types" hasIcon>Type</HelpLink> | Description                                                                                                     |
| ---------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |

> | **type**   | <HelpLink href="#api-basics/types">Enum</HelpLink>         | The type of the output. It can either be `lambda` or `file`.                                                    |

Reviewer: ```suggestion
| **type**   | <HelpLink href="#api-basics/types">Enum</HelpLink>         | The type of the output. Either `lambda` or `file`.                                                    |
```

> | **path**   | <HelpLink href="#api-basics/types">String</HelpLink>       | The absolute path of the file or serverless function.                                                           |

Reviewer: ```suggestion
| **path**   | <HelpLink href="#api-basics/types">String</HelpLink>       | The absolute path of the file or Serverless Function.                                                           |
```

| **digest** | <HelpLink href="#api-basics/types">String</HelpLink>       | The SHA1 of the file.                                                                                           |
| **mode**   | <HelpLink href="#api-basics/types">Number</HelpLink>       | The POSIX file permissions.                                                                                     |
| **size**   | <HelpLink href="#api-basics/types">Number</HelpLink>       | The size of the file in bytes.                                                                                  |

> | **lambda** | <HelpLink href="#api-basics/types">Map</HelpLink>          | If the output is a serverless function it returns an object that contains `functionName` and `deployedTo` keys. |

Reviewer: ```suggestion
| **lambda** | <HelpLink href="#api-basics/types">Map</HelpLink>          | If the output is a Serverless Function, it returns an object that contains `functionName` and `deployedTo` keys. |
```
